handlers/utils/compute_utils.go

```diff
@@ -0,0 +1,1186 @@
+package utils
```

Comment:
Many of the functions in here don't appear to be util functions. They appear to be specific to compute logic. Only put things in a util package if they are used across multiple packages. Otherwise, just include them in the same package as where they're used and unexport (lowercase) them.